oci/private/pull.bzl

```diff
@@ -272,6 +272,11 @@ oci_pull = repository_rule(
 )
 
 _MULTI_PLATFORM_IMAGE_ALIAS_TMPL = """\
+exports_files(
```

Comment:
can we make this a filegroup(name = "digest") instead?

Reply:
Making it a `filegroup` may make it harder to be used within repository rule context. Is there any problem with it being a file instead?

Reply:
I'd imagine this target being only used in the build rule context, may i ask the reason why just reading the index.json file created oci_pull isn't an option?

I may be paranoid, but my concerns 

- This wouldn't (probably) work with RBE
- This is a file laying around in the repository directory, easy to break and easy to accidentally override.
- It's not likely to work for a confused user because of `alias` created by oci_alias target

Reply:
Fixed this to use a `filegroup`, it works for my usecase.